docs/docs/guides/tools/saved-views.md

```diff
@@ -0,0 +1,187 @@
+
+```python
+view.to_grid().show()
+```
+
+### Set displayed columns
+
+Use `.set_columns()` to set the columns to be displayed in the view. Specify one or more columns to be displayed.
+
+```python
+view.set_columns("id", "op_name")
+```
+
+### Add columns
+
+Use `.add_column()` to add one or more new columns to the view. Specify one or more columns to be added.
+
+```python
+view.add_column("started_at", "Created")
+```
+
+### Sort columns
+
+Use `.sort_by()` to sort results based on a specific column. Specify the column name to be sorted and the sort order (`asc` or `desc`).
+
+```python
+view.sort_by("started_at", "desc")
+```
+
+### Filter by operation name
+
+In Weave, every trace or eval is associated with an operation name.
+Use `.filter_op()` to filter the `SavedView` to only include calls where that specific operation was executed. 
+
+```python
+view.filter_op("Evaluation.predict_and_score")
+```
+
+:::tip
+`.filter_op()` is a shortcut for filtering by operation name. It ensures the correct URI format is used automatically. Alternatively, you can achieve the same result with `.add_filter("op_name", "equals", <operation_name>)`.
+:::
+
+### Filter by operator and condition
+
+Use `.add_filter()` to apply a custom filter to the view. Define the filter using one of the [supported filter operators](#filter-operators) and a condition.
+
+```python
+view.add_filter("output.model_latency", ">=", 5)
+```
+
+#### Filter operators
+
+| Operator | Description | Example |
+|----------|-------------|---------|
+| `"contains"` | Checks if a string contains a substring. | `view.add_filter("output.status", "contains", "error")` |
+| `"equals"` | Checks if a string is exactly equal to a given value. | `view.add_filter("user.name", "equals", "Alice")` |
+| `"in"` | Checks if a string is in a list of values. | `view.add_filter("category", "in", ["A", "B", "C"])` |
+| `"="` | Checks if a number is equal to a value. | `view.add_filter("output.score", "=", 80)` |
+| `"≠", "!="` | Checks if a number is not equal to a value. | `view.add_filter("metrics.loss", "!=", 0.5)` |
+| `"<"` | Checks if a number is less than a value. | `view.add_filter("age", "<", 30)` |
+| `"≤", "<="` | Checks if a number is less than or equal to a value. | `view.add_filter("metric.value", "<=", 100)` |
+| `">"` | Checks if a number is greater than a value. | `view.add_filter("output.score", ">", 90)` |
+| `"≥", ">="` | Checks if a number is greater than or equal to a value. | `view.add_filter("output.model_latency", ">=", 5)` |
+| `"is"` | Checks if a boolean field is `True` or `False`. | `view.add_filter("is_active", "is", True)` |
+| `"after"` | Checks if a date is after a given timestamp. | `view.add_filter("started_at", "after", "2024-01-01")` |
+| `"before"` | Checks if a date is before a given timestamp. | `view.add_filter("ended_at", "before", "2024-12-31")` |
+| `"is empty"` | Checks if a field is empty (`None` or `""`). | `view.add_filter("comments", "is empty", None)` |
```

Comment:
I believe your doc is correct that the None value is required in this case but I should probably make it optional.